.travis.yml

```diff
@@ -13,7 +13,7 @@ addons:
 before_install:
   - if test "$TRAVIS_OS_NAME" == "osx"; then brew update; brew install icu4c; fi
 env:
-  - KOREBUILD_DNU_RESTORE_CORECLR=true
+  - KOREBUILD_DNU_RESTORE_CORECLR=true MONO_THREADS_PER_CPU=2000 KOREBUILD_TEST_DNXCORE=true
```

Comment:
Actually, https://docs.travis-ci.com/user/environment-variables/#Defining-Multiple-Variables-per-Item is more pertinent since we're _avoiding_ matrix variables i.e. additional builds.